Lib/_colorize.py

```diff
@@ -170,6 +170,9 @@ class Argparse(ThemeSection):
     label: str = ANSIColors.BOLD_YELLOW
     action: str = ANSIColors.BOLD_GREEN
     reset: str = ANSIColors.RESET
+    error: str = ANSIColors.BOLD_MAGENTA
+    warning: str = ANSIColors.BOLD_MAGENTA
```

Comment:
```suggestion
    warning: str = ANSIColors.BOLD_YELLOW
```

Perhaps different for error and waring?

<img width="674" height="158" alt="image" src="https://github.com/user-attachments/assets/fd7bf944-b7a8-4268-8f46-489dc95571b5" />